.github/ISSUE_TEMPLATE/25_api_proposal.md

```diff
@@ -0,0 +1,53 @@
+---
+name: API proposal
+about: Propose a change to the public API surface
+title: ''
+labels: api-suggestion
+assignees: ''
+
+---
+
+## Background and Motivation
+
+<!--
+We welcome API proposals! We have a process to evaluate the value and shape of new API. There is an overview of our process [here](https://github.com/dotnet/aspnetcore/blob/main/docs/APIReviewProcess.md). This template will help us gather the information we need to start the review process.
+First, please describe the purpose and value of the new API here.
+-->
+
+## Proposed API
```

Comment:
Since you used `Alternative Designs` verbiage, should this be `Proposed Design`?

Reply:
Thanks for your comment, @ryan-dayrit.
This is actually about the other designs the owner of the proposal may have considered before proposing this design. Hence, this phrase seems valid in this context.